### ISSUE TITLE
Show full details of activity (pictures, description, etc)

### ISSUE THREAD

**Issue by Jmerrey** [AppCreativity/Kliva#13]

**Comment by Depechie** [comment]
Jmerry could you maybe give a bit more detail on what you currently lack on an activity page?
We have added pictures already, but due to the Strava api limitations, we can only show those of your own activites :(

**Comment by Jmerrey** [comment]
I have an activity with pictures uploaded, but I can't see the pictures. They are visible on the ios app. 

-----Original Message-----

**Comment by Depechie** [comment]
Hmm weird, could you paste a screenshot of the activity detail page on the Photos pivot?
And maybe tell me on what version of the app you are on ( you can see this on the settings page ).

**Comment by Jmerrey** [comment]
The app version is 1.3.2.  There isn't a photos pivot in the activity details page. I've attached a screenshot. 

-----Original Message-----

**Comment by Jmerrey** [comment]
As a follow up, I uninstalled and reinstalled the app. When I went to the activity details page that I have photos uploaded to, the photos pivot appeared for just a second and then went away. 

-----Original Message-----
From: "John Merrey" jmerrey@gmail.com
Sent: ‎5/‎19/‎2015 10:28 AM
To: "AppCreativity/Kliva" reply@reply.github.com; "AppCreativity/Kliva" Kliva@noreply.github.com
Subject: RE: [Kliva] Show full details of activity (pictures, description,etc)(#13)

The app version is 1.3.2.  There isn't a photos pivot in the activity details page. I've attached a screenshot.
